app/src/components/Image.tsx

```diff
@@ -6,6 +6,8 @@ class Image extends EnhancedComponent<IImageProps, IImageState> {
 	public static defaultProps: IImageProps = {
 		...EnhancedComponent.defaultProps,
 		path: "/logo.png",
+		width: 132,
+		height: 74,
```

Comment:
I think this width and height can be larger.

For example, on YouTube, the size of the thumbnail is 270x150 when the icon size is 24x24. Since our icons are 30x30, we should make this approx 300x170px (16:9 aspect ratio - standard for YouTube thumbnail)

Accordingly you can also increase font size to 14px or 16px